lib/analyze_solution.rb

```diff
@@ -6,11 +6,10 @@ class AnalyzeSolution
   def call
     code_to_analyze = File.read(solution_path / FILENAMES[exercise_slug])
 
-    puts "Analysing #{exercise_slug}"
-    classified_exercise = exercise_slug.tr('-', '_').classify
```

Comment:
While running the tests, I uncovered a bug where the 'classify' method was stripping the 's' off the end of 'HighScores' because the method was created to singularize Rails ActiveRecord table names into model names. This caused the analyzer to fail.